docs/develop/manifest-element-ordering.md

```diff
@@ -6,17 +6,19 @@ ms.date: 11/16/2018
 
 # How to find the proper order of manifest elements
 
-The XML elements in the manifest of an Office Add-in must not only be under to the proper parent element; they must be in a specific order, relative to each other, under the parent. For example, In the `<OfficeApp>` element, the `<Id>`, `<Version>`, `<ProviderName>` must appear in that order. If an `<AlternateId>` element is added, it must be between the `<ID>` and `<Version>` element. If any element is in the wrong order, your manifest will not be valid and your add-in will not load.
+The XML elements in the manifest of an Office Add-in must not only be under the proper parent element; they must be in a specific order, relative to each other, under the parent.
```

Comment:
Nit: I keep getting tripped up by the "not". Consider:
"The XML elements in the manifest of an Office Add-in must be under the proper parent element **and** in a specific order, relative to each other, under the parent."